manifest.json

```diff
@@ -28,6 +28,7 @@
         "third-party/echarts.simple.min.js",
         "common/content-script/util.js",
         "common/content-script/hypertrons-dashboard.js",
+        "common/content-script/simple-graph.js",
```

Comment:
You may need to move it up to `hypertrson-dashboard.js`, otherwise main class may not find the declaration I think, need to check.

Reply:
Yeah, I have checked that and it can work well.

![image](https://user-images.githubusercontent.com/26642280/83004697-8f426400-a042-11ea-829b-e34bdad828c6.png)